tools/flub/.mocharc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Probably not for this PR, but do we have a shared base mocha config? Seems like we could reduce some redundancy and ensure consistency via shared settings.

Reply:
We do have a [shared mocha config](https://github.com/microsoft/FluidFramework/tree/main/packages/test/mocha-test-setup), but I don't know that it's that useful here since these tests are very different from our usual tests. But I also don't know much about mocha. :)

Reply:
That's fair. Might be worth an audit at some point in the future, but I definitely don't want to stall this PR or anything :)